pennylane_orquestra/_version.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Version information.
+   Version number (major.minor.patch[-label])
+"""
+
+
+__version__ = "0.0.13-dev"
```

Comment:
nit: perhaps start with version 0.0.1-dev?

Reply:
Good point! On one hand, this points out a correction that needs to be made for sure (this would need to be `"0.13.0-dev").

Previously with brand new plugins we actually kind of "bumped" to the current PennyLane version, so the idea here was that releasing this would give version `0.13.0`. But agreed, `0.1.0-dev` or so makes actually more sense. @josh146 any preferences there?

Reply:
Hey, yep! This is deliberate. While we are still prior to version 1.0 of PennyLane, 'anything goes' (as per semver).

Since we make lots of breaking changes still, we found it is easier to keep track of the plugins if they all roughly keep in step with the PL version number.